python/cudf/cudf/core/column/column.py

```diff
@@ -973,7 +975,7 @@ def as_string_column(
 
     def as_decimal_column(
```

Comment:
Would we want to split `as_decimal_column` off into `as_decimal32_column` and `as_decimal64_column` where it's implemented?